tests/migration/online_migration/register_system.pm

```diff
@@ -15,6 +15,7 @@ use migration;
 
 sub run {
     select_console 'root-console';
+    set_var('SCC_ADDONS', get_var('SCC_ADDONS_2')) if (get_var('SCC_ADDONS_2'));
```

Comment:
Can you comment what 'SCC_ADDONS_2' used for?

Reply:
we should use SCC_ADDONS, and have a module that swap the values, the module will have documentation to explain why, otherwise questions like above would arise.